juju/client/connector.py

```diff
@@ -0,0 +1,121 @@
+import asyncio
+import logging
+import macaroonbakery.httpbakery as httpbakery
```

Comment:
This goes to a second import block of external libraries.

Reply:
done